activity.py

```diff
@@ -42,37 +41,34 @@ def build_toolbar(self):
         toolbar_box.toolbar.insert(activity_button, -1)
         activity_button.show()
 
-        separator = Gtk.SeparatorToolItem()
-        separator.props.draw = False
-        separator.set_expand(False)
-        toolbar_box.toolbar.insert(separator, -1)
-        separator.show()
+        self._add_toolbar_separator(toolbar_box)
```

Comment:
This change would be better done in sugar-toolkit-gtk3 sugar3/activity/widgets.py